GoogleForCreators/web-stories-wp
# Always display the design space guideline when an element is being transformed

## Summary


## Relevant Technical Choices



## To-do



## User-facing changes
The guideline now appears when doing any transformations with an element:
![guideline](https://user-images.githubusercontent.com/3294597/112311220-c656c580-8ca5-11eb-890b-e4234f4f95b7.gif)



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Drag a shape from the shape library, verify the design space guideline is displayed while the element is being dragged
2. Verify the guideline disappears after finishing dragging.
3. Now move, resize, rotate the shape.
4. Verify that during these activities, the design space guideline is displayed again, even when not snapping to it.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6367 
FIxes #5134
